.github/workflows-src/steps.lib.yml

```diff
@@ -102,7 +102,11 @@ name: "Critical Integration Tests"
 shell: bash
 run: |
   export SHELL=''
+  set +e
   go test -timeout 10m github.com/ActiveState/cli/test/integration -test.v -test.run "^TestLanguageMsiActivePerl|TestDeployIntegrationTestSuite|TestInitIntegrationTestSuite|TestPushIntegrationTestSuite|TestUpdateIntegrationTestSuite|PrepareIntegrationTestSuite$"
+  testCode=$?
+  head -n-0 build/*log 2>/dev/null
+  exit ${testCode}
```

Comment:
Doesn't `set +e` make this redundant?

ie. the only change required is `set +e` and `cat build/*.log` at the end

Reply:
We want the success/failure to be solely determined by the `go test` results; The `head` call will throw off the value of `$?`, so we have to explicitly set the exit code.